### PR TITLE
Fix use lure at break and add optional animation time

### PR DESCRIPTION
**Pull request by MTK4355**

### CODE REVIEW

File: PokemonGo.RocketAPI.Console/GUI.cs [hunk 1]
```diff
@@ -1370,6 +1391,11 @@ private void checkBox20_CheckedChanged(object sender, EventArgs e)
             Globals.BreakAtLure = checkBox20.Checked;
         }
 
+        private void checkBox24_CheckedChanged(object sender, EventArgs e)
```
Comment (SscSPs) [inline PokemonGo.RocketAPI.Console/GUI.cs]: What is this one? just curious.

**Comment by MTK4355** [comment]
That is the toggle to not use animation times.

File: PokemonGo.RocketAPI.Console/GUI.cs (hunk 1 above)
Comment (MTK4355) [inline PokemonGo.RocketAPI.Console/GUI.cs]: Toggle to use or ignore animation times at evolve.

Comment (MTK4355) [inline PokemonGo.RocketAPI.Console/GUI.cs]: Thanks for the head's up. I must have accidentally deleted the changed event logic before committing!

**Pull request merged**